Doc/includes/email-mime.py

```diff
@@ -22,7 +19,7 @@
     with open(file, 'rb') as fp:
         img_data = fp.read()
     msg.add_attachment(img_data, maintype='image',
-                                 subtype=imghdr.what(None, img_data))
+                                 subtype='jpeg')
```

Comment:
```suggestion
                                 subtype='png')
```

A few lines above there is `for file in pngfiles:`…

Reply:
Since this PR is already merged, I'll add this to my clean up branch at #91520

Reply:
👍🏻